R/timeNumeric.R

```diff
@@ -0,0 +1,49 @@
+#' Time field provided in character as "%H:%M:%S" or in Period class ("H M S")
```

Comment:
Is this really `character`, rather than `Date`? Visual appearance is the same but the Date objects have time handling tools, characters do not.